### PR TITLE
[risk=low] using actual less than or equal to unicode symbol in chart labels

### PR DESCRIPTION
**Pull request by SrushtiGangireddy**
1. Changing <= 20 to use actual &#8804 unicode symbol.

### CODE REVIEW

File: public-ui/src/app/data-browser/chart/chart.component.ts [hunk 1]
```diff
@@ -168,10 +168,11 @@ export class ChartComponent implements OnChanges, AfterViewInit {
             const label = this.axis.defaultLabelFormatter.call(this);
             // Change <= 20 count to display '<= 20'
             if (label <= 20) {
-              return '<= 20';
+              return '&#8804; 20';
```
Comment (SrushtiGangireddy) [inline public-ui/src/app/data-browser/chart/chart.component.ts:171]: using less than or equal to unicode.

File: public-ui/src/app/views/ehr-view/ehr-view.component.html [hunk 2]
```diff
@@ -101,8 +101,21 @@ <h6 *ngIf="searchText.value.length > 0" class="primary-display medline-link">Int
...
                       participant
                       counts are approximate to protect participant privacy. Participant counts are rounded to the
-                      nearest multiple of 20. Participant counts of 0 - 20 are returned as <= 20</div>
-                  </clr-tooltip-content> </clr-tooltip> </div> <div class="tbl-d  body-lead info-text">
+                      nearest multiple of 20. Participant counts of 0 - 20 are returned as <= 20
+                    </div>
+                  </clr-tooltip-content>
+                </clr-tooltip>
+              </div>
+              <div class="tbl-d body-lead info-text"> % of Total Participants
```
Comment (SrushtiGangireddy) [inline public-ui/src/app/views/ehr-view/ehr-view.component.html:109]: https://precisionmedicineinitiative.atlassian.net/browse/DB-356
Adding participant percentage.

File: public-ui/src/app/views/ehr-view/ehr-view.component.html [hunk 3]
```diff
@@ -166,8 +179,16 @@ <h6 *ngIf="searchText.value.length > 0" class="primary-display medline-link">Int
                 </div>
               </ng-template>
               <div class="tbl-r-group">
+                <div class="body-lead info-text tbl-d" *ngIf="checkCount(r.countValue); else Non20Counts;">
+                  &le; {{r.countValue | number}}
```
Comment (SrushtiGangireddy) [inline public-ui/src/app/views/ehr-view/ehr-view.component.html:183]: https://precisionmedicineinitiative.atlassian.net/secure/RapidBoard.jspa?rapidView=22&projectKey=DB&modal=detail&selectedIssue=DB-355

Converting this to number adds comma.

**Review by mhigbyflowers — APPROVED** [review]
The symbol looks a lot better. Thanks

**Pull request merged**